packages/plugins/Export/src/store/actions.ts

```diff
@@ -1,7 +1,92 @@
 import { jsPDF } from 'jspdf'
 import { PolarActionTree } from '@polar/lib-custom-types'
+import { Map } from 'ol'
 import { ExportFormat, ExportGetters, ExportState } from '../types'
 
+// PDF options
+const dims = {
+  a0: [1189, 841],
+  a1: [841, 594],
+  a2: [594, 420],
+  a3: [420, 297],
+  a4: [297, 210],
+  a5: [210, 148],
+}
+
+// Screenshot canvas
+const CANVAS_ID = 'export-canvas'
+
+const convertToPdf = (map: Map, src: string, size: number[]) => {
+  // NOTE: when supporting more formats, scale map accordingly
+  const format = 'a4'
+  const dim = dims[format]
+  // Import of jspdf is in mounted.
+  const jsPdf = new jsPDF('landscape', undefined, format) // eslint-disable-line
+  jsPdf.addImage(src, 'JPEG', 0, 0, dim[0], dim[1])
+  // Reset original map size
+  map.setSize(size)
+  map.getView().setResolution(map.getView().getResolution())
```

Comment:
There must've been a reason for the variable beforehand. Are you sure, this doesn't change anything? `map.getSize()` is still set as a variable on the other hand.

Now that I think about it: What happens if neither the size of the map or the resolution is reset?

Reply:
Upon revisiting this, yeah, it doesn't change anything. In fact, we can even skip the line altogether. I've dropped the whole map view modification thing in https://github.com/Dataport/polar/pull/180/commits/b0edcaf1b2475f2c298169a1d0abbcb611f5de76.

After all, size and resolution of the map are never changed anyway. It's the same on `main`.